src/main/java/com/scalefocus/blog_api/mapper/BlogMapper.java

```diff
@@ -0,0 +1,31 @@
+package com.scalefocus.blog_api.mapper;
+
+
+import com.scalefocus.blog_api.dto.BlogDto;
+import com.scalefocus.blog_api.entity.Blog;
+
+public class BlogMapper {
+
+    public static BlogDto mapToBlogDto(Blog blog) {
+
+        return new BlogDto(
+                blog.getId(),
```

Comment:
you can use @Builder on BlogDto and here use the .builderAproach

Reply:
Done